package.json

```diff
@@ -80,11 +80,14 @@
     "details-element-polyfill": "^2.4.0",
     "dialog-polyfill": "^0.5.6",
     "dotenv": "^16.0.3",
+    "eslint": "8.25.0",
+    "eslint-config-prettier": "8.5.0",
```

Comment:
we don't need `eslint`, as we even already have it included via `xo`

Reply:
fixed